.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+indent_style = space
+indent_size = 4
```

Comment:
This looks like an editor specific file that I'd prefer we don't add to the repository.

Reply:
This is actually not an editor-specific file, but one that is designed so that it can be read and supported by almost any editor. Many major IDEs and some of the more lightweight editors already support this directly or have a plugin for it. See: https://editorconfig.org/ -- anyway, your call.

Reply:
Ahh, very good!  I was obviously unaware of that.  Sometime some comments added to it to explain what it is for and the Web URL will be a good idea for others that are curious.  I've no problem keeping it in the patch set as it appears useful.